dandiapi/api/models/asset_paths.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from django.db import models
+from django.db.models import Q
+from django_extensions.db.models import TimeStampedModel
+
+from dandiapi.api.models.asset import Asset
+from dandiapi.api.models.version import Version
+
+
+class AssetPath(TimeStampedModel):
```

Comment:
What is the reasoning for using `TimeStampedModel` on these models? If we want to consider the modification of an `AssetPath` or `AssetPathRelation` as a modification of the `Asset` itself, I wonder if we should just directly update `AssetPath.asset.modified`. In general, I worry about introducing more unnecessary timestamps into the system considering how messy that aspect of the system is currently.

Reply:
I agree, this predated our discussion regarding `TimeStampedModel`. I'll change it.

Reply:
https://github.com/dandi/dandi-archive/pull/1312/commits/9ee70ad4ffc9af64008bcae5f57073e4477cafed